classes/memos.rb

```diff
@@ -0,0 +1,35 @@
+require 'pstore'
+
+class Memos
+  def self.all
+    store = PStore.new('./tmp/storage.json')
+    store.transaction do
+      store['memos'] || {}
+    end
+  end
+
+  def self.find(id)
+    store = PStore.new('./tmp/storage.json')
+    store.transaction do
+      memos = store['memos'] || {}
+      memos.select { |k| k == id }
+    end
+  end
+
+  def self.add(memo)
+    store = PStore.new('./tmp/storage.json')
+    store.transaction do
+      memos = store['memos']
+      memos = {} if memos.nil?
```

Comment:
これは `all` メソッドでやっているのと同じように 1 行にまとめられるかも？

```suggestion
      memos = store['memos'] || {}
```

Reply:
確かにそうですね！こちら修正しました！